### DISCUSSION TITLE
Filing date alternatives

### DISCUSSION THREAD

**Discussion by Eran-T** [freelawproject/courtlistener#3319] [Q&A] [answered]
Hi,

I noticed that many of the dockets only have a 'date_created' and a 'date_modified' field. 
What would be the best alternative (if there are any) to get an approximation of the filing date? Is there a field in the clusters/sub-opinions that could maybe help with this goal? 

Thanks!

**Comment by mlissner** [comment, marked as answer]
The date_filed field on the cluster is usually the best field for that. Is it possible you didn't see it?

**Comment by Eran-T** [comment]
Hi.
You are right. I'll use it when available.

thanks for the help!

**Reply by mlissner** [reply]
Sorry, I just realized a subtlety here. If you're looking at opinions, you'll want the cluster.date_filed field. If you're looking at cases, you'll want the docket.date_filed field. 

If it's clusters, note also that there's a field, `date_is_approximate` that affects predominantly older cases. Historically, some cases didn't have actual dates, and just said something like "Fall Term, 1823" or "October, 1823". In those instances we need a date_filed value for sorting, etc, but it's approximate, so it's worth labeling it as such in your interface.

**Reply by Eran-T** [reply]
Hi, just noticed your reply.
Thanks for the update. 
does this mean that there is no alternative (referring to the original question)?

**Reply by mlissner** [reply]
Hm, can you give an example, so I can take a look?

**Reply by Eran-T** [reply]
Sure, for example here: https://www.courtlistener.com/api/rest/v3/dockets/64502274/
there is no date_filed field. My question is, whether or not there is a way to approximate the date_filed field using the API (for example using the earliest date_filed from the clusters). 
Thanks!

**Reply by mlissner** [reply]
OK, so looking at this docket in the front end:

https://www.courtlistener.com/docket/64502274/d/

I see what you mean. There's no date filed field, but it links to an opinion:

https://www.courtlistener.com/opinion/7501287/ordono-v-quintana/?

So that's the date the *opinion* was filed. 

> My question is, whether or not there is a way to approximate the date_filed field using the API

I take this to mean: If you have the date of the opinion, but not the date of the docket, can you somehow approximate the date of the docket? I'm not sure I have good ideas for that. Some cases can take years, others can take hours (though I suppose you wouldn't get a decision that quickly normally). 

So....if you only know the date of the decision, I don't think you can ever guess the date of the case, without getting the full docket, which can be very hard in non-federal cases. (Federal ones can use our Fetch API.)

Nobody has ever asked for this before. Am I understanding your question correctly?
